web/client/utils/MapInfoUtils.js

```diff
@@ -152,12 +152,16 @@ const MapInfoUtils = {
         const center = {x: lngCorrected, y: props.point.latlng.lat};
         let centerProjected = CoordinatesUtils.reproject(center, 'EPSG:4326', props.map.projection);
         let bounds = MapInfoUtils.getProjectedBBox(centerProjected, resolution, rotation, size, null);
+        let queryLayers = layer.name;
+        if (layer.queryable) {
```

Comment:
Is the queryable attribute of layers something custom to your application? I get it's an array of layers to be queried instead of the original layer. I think queryable is a misleading name. Usually queryable is a flag to distinguish layers that can be queries.